shuup/front/template_helpers/urls.py

```diff
@@ -7,15 +7,24 @@
 # LICENSE file in the root directory of this source tree.
 from __future__ import unicode_literals
 
+from django.conf import settings
 from django.core.urlresolvers import NoReverseMatch, reverse
 from jinja2 import contextfunction
 
 from shuup.core.models import Category, Product
+from shuup.utils.importing import cached_load
 
 
 @contextfunction
 def model_url(context, model, absolute=False):
     uri = None
+
+    model_url_method = cached_load('SHUUP_MODEL_URL_RESOLVER_SPEC')
```

Comment:
Put this setting also in `shuup/front/settings.py` and document it. Move the original implementation to some other place like `shuup/front/utils/urls.py` and set this function as the `SHUUP_MODEL_URL_RESOLVER_SPEC` default value

Reply:
I create new PR with fixes. Tests passes.